src/ontology/phenio-odk.yaml

```diff
@@ -25,24 +22,38 @@ import_group:
   slme_individuals: exclude
   mirror_max_time_download: 400
   products:
-    - id: nbo
-      use_base: TRUE
+    - id: bfo
+    - id: caro
+      make_base: TRUE
+    - id: chebi
+      is_large: TRUE
+      use_gzipped: TRUE
+      make_base: TRUE
+    - id: eco
+      make_base: TRUE
     - id: fao
       make_base: TRUE
-    - id: oba
-      use_base: TRUE
-    - id: stato
-    - id: bfo
+    - id: go
+      make_base: TRUE
     - id: hsapdv
       make_base: TRUE
     - id: mpath
       make_base: TRUE
+    - id: nbo
+      make_base: TRUE
     - id: ncbitaxon-taxslim
       mirror_from: http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl
-    - id: caro
+      make_base: TRUE
+    - id: oba
+      make_base: TRUE
+    - id: pato
+      make_base: TRUE
+    - id: ro
+      make_base: TRUE
```

Comment:
In general its better not do this for RO, use ro-base. The reason is that RO has a few namespaces like BFO (part of) and RO_HOM which we don't want to prune away. Also, there are some axioms in RO that are non-base but we still need them.